tools/docker/Dockerfile

```diff
@@ -20,12 +20,15 @@ RUN DNF=dnf \
     gcc \
     gettext \
     git-core \
-    java-17-openjdk-headless" \
+    java-17-openjdk-headless \
+    supervisor" \
+    && $DNF -y install  https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm \
     && $DNF -y install dnf-plugins-core \
     && $DNF -y config-manager --set-enabled crb \
     && $DNF -y install $INSTALL_PACKAGES \
     && $DNF -y clean all \
     && rm -rf /var/cache/dnf
+COPY ./tools/docker/supervisord/supervisord.conf /etc/supervisord.conf
```

Comment:
I'm concerned about the potential approach that this might be taking if are going to run supervisor in the container image. 
Running multiple processes in the same container is not considered a good practice and in EDA we have tried to avoid it from the beginning.

Reply:
In gateway and in EDA, supervisor is used to run rsyslog.

Trying to follow their approach here to lift and shift these requirements.

Do you have alternative suggestions about a different approach?

Reply:
After further thought and discussion, I will refactor this to rework this to a simpler approach